### ISSUE TITLE
Security: Avoid running code for static parser

### ISSUE THREAD

**Issue by de-code** [ImperialCollegeLondon/R2T2#57]
The static parser contains an `eval` statement. In general it's probably good to use `eval` as little as possible.
I could see `r2t2` being used on untrusted source to find references. Having `eval` may run arbitrary code (or even if the syntax parsed doesn't allow it, it would be easier to affirm that this is not the case).

**Comment by dalonsoa** [comment]
Yes, probably that one can be avoided. We can look for a workaround that.
